StRoot/StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -1853,10 +1868,8 @@ void StPicoDstMaker::fillEvent() {
   StZdcTriggerDetector &ZDC = ev->zdcTriggerDetector();
   picoEvent->setZdcSumAdcEast( ZDC.adcSum(east) );
   picoEvent->setZdcSumAdcWest( ZDC.adcSum(west) );
-  if ( ev->triggerData() ) {
-    picoEvent->setZdcUnAttenuatedEast( ev->triggerData()->zdcUnAttenuated(east) );
-    picoEvent->setZdcUnAttenuatedWest( ev->triggerData()->zdcUnAttenuated(west) );
-  }
+  picoEvent->setZdcUnAttenuatedEast( ev->triggerData()->zdcUnAttenuated(east) );
+  picoEvent->setZdcUnAttenuatedWest( ev->triggerData()->zdcUnAttenuated(west) );
```

Comment:
Could you please return the if statement back? It is needed as a protection for the simulation.

Reply:
Before I take the steps of adding this back, I want to explain my understanding of what I think is happening.

While I was making my edits to StPicoDstMaker.cxx, pull request #29 was approved and merged. This meant that the version of the StPicoDstMaker.cxx that I was editing had new edits in the star-sw repo that my version did not have.

When I first submitted this pull request, I noticed the new edits from merging pull request #29. I saw that the edits were quick to make, so I added them. However, that was not a clean way of doing things. So, @plexoos had a suggestion.

His suggestion was to update my pull request and rebase my commits to commit #28. If I am understanding this properly, that means my commits will will be merged with however the repo looked liked once commit #28 was done, which also means, that I did not have to worry about the commits that happened after #28.

After understanding @plexoos suggestion, I did what he suggested. Admittedly, rebasing is something I have not used before, so there is a chance that I did not do it properly. Please, let me know if that is the case.

With all of this said, if the code owners agree that I should go ahead and add the edits done by commit number #29 and updated the pull request, I would be more than happy to do so. I just wanted to explain what is going on to avoid doing something that would conflict with what @plexoos  suggested.

Reply:
Bassam, you did rebase your branch but you did not delete the commits. Maybe this answer can explain better: https://stackoverflow.com/questions/1338728/delete-commits-from-a-branch-in-git/1338758#1338758

There is a tutorial https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase

Reply:
Hi @plexoos! When I do `git rebase -i 16bad14f1e2ca98b3942d8dd37dbb8569f03d293` , I get the following empty window where none of my commits are shown. So, I can't edit/remove any of them.

<img width="725" alt="Screen Shot 2021-06-14 at 11 41 31 AM" src="https://user-images.githubusercontent.com/85306311/121928472-ce069300-cd05-11eb-80ff-94a8c6ddd09e.png">

Reply:
Show us the output of `git show -q`
You need to make sure that your `main` branch points to b229a989ad7741e86d49328ce6cc2127f86a86f7

Reply:
Attached is a screenshot after running the command above:

<img width="803" alt="Screen Shot 2021-06-14 at 12 15 04 PM" src="https://user-images.githubusercontent.com/85306311/121932246-453e2600-cd0a-11eb-8b81-f6c3e35bfb9b.png">

Reply:
Yes, this looks as expected. I don't know why your text editor does not display any entries... For me it looks like this when I do `git rebase -i 16bad14f`
![Screen Shot 2021-06-14 at 1 19 10 PM](https://user-images.githubusercontent.com/5005079/121932940-6fe0ac80-cd13-11eb-8f1a-12edc2c1b03c.png)

If you want I can clean up this branch for you, just let me know

Reply:
Your screenshot is what I was expecting to see based on tutorials that I have been trying to follow. I am not sure why my text editor is not displaying these entries either. My plan was to use `squash` or `fixup` to condense the number of commits to clean up the pull request. If it is not any trouble for you, yes, you can clean up this branch for me. Thank you very much.